src/components/InstantSearch.js

```diff
@@ -97,7 +97,9 @@ export default createInstantSearchComponent({
           [this.suit('', 'ssr')]: false,
         },
       },
-      isVue3 ? this.$slots.default() : this.$slots.default
+      isVue3
+        ? this.$slots.default && this.$slots.default()
+        : this.$slots.default
```

Comment:
maybe we should have a shared util for this?

Reply:
Makes sense. I'll see what I can do in a separate PR.